### PR TITLE
Add spec_url for `SVGAnimationElement`

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Related issues

Fixes https://github.com/mdn/browser-compat-data/issues/25384

### CODE REVIEW

File: api/SVGAnimationElement.json [hunk 1]
```diff
@@ -329,6 +333,7 @@
       },
       "getCurrentTime": {
         "__compat": {
+          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getCurrentTime",
```
Comment (yashrajbharti) [inline api/SVGAnimationElement.json:336]:
```suggestion
           "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SVGAnimationElement/getCurrentTime",
          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getCurrentTime",
```

File: api/SVGAnimationElement.json [hunk 2]
```diff
@@ -371,6 +376,7 @@
       },
       "getSimpleDuration": {
         "__compat": {
+          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getSimpleDuration",
```
Comment (yashrajbharti) [inline api/SVGAnimationElement.json:379]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SVGAnimationElement/getSimpleDuration",
          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getSimpleDuration",
```

File: api/SVGAnimationElement.json [hunk 3]
```diff
@@ -413,6 +419,7 @@
       },
       "getStartTime": {
         "__compat": {
+          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getStartTime",
```
Comment (yashrajbharti) [inline api/SVGAnimationElement.json:422]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SVGAnimationElement/getStartTime",
          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__getStartTime",
```

File: api/SVGAnimationElement.json [hunk 4]
```diff
@@ -188,6 +190,7 @@
       },
       "endElement": {
         "__compat": {
+          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__endElement",
```
Comment (yashrajbharti) [inline api/SVGAnimationElement.json:193]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SVGAnimationElement/endElement",
          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__endElement",
```

File: api/SVGAnimationElement.json [hunk 5]
```diff
@@ -47,6 +47,7 @@
       },
       "beginElement": {
         "__compat": {
+          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__beginElement",
```
Comment (yashrajbharti) [inline api/SVGAnimationElement.json:50]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SVGAnimationElement/beginElement",
          "spec_url": "https://svgwg.org/specs/animations/#__svg__SVGAnimationElement__beginElement",
```

**Review by yashrajbharti — CHANGES_REQUESTED** [review]
The "mdn_url" of the respective pages can be added to.
The MDN feature pages are in pr [here](https://github.com/mdn/content/pull/37180) and [here](https://github.com/mdn/content/pull/37182) which is why I raised the issue.

**Comment by caugner** [comment]
> The "mdn_url" of the respective pages can be added to. The MDN feature pages are in pr [here](https://github.com/mdn/content/pull/37180) and [here](https://github.com/mdn/content/pull/37182) which is why I raised the issue.

The mdn_urls will get added automatically once the pages referencing these BCD keys landed in content, so let's avoid adding those links before the pages exist.

**Review by caugner — APPROVED** [review]

**Comment by yashrajbharti** [comment]
Thanks!

**Pull request merged**